src/components/Auth/PrivateRoute.js

```diff
@@ -12,7 +11,7 @@ const PrivateRoute = ({component: Component, ...rest}) => {
     }, []);
 
     const getUser = async () => {
-        const msg = await getUserEmail();
+        const msg = localStorage.getItem("email");
```

Comment:
If I change this local storage item to someone else's email address? Will I be able to see their data?